src/Accelerators/NNPA/Runtime/zDNNExtension/MatMul.c

```diff
@@ -77,6 +92,11 @@ static zdnn_status zdnn_matmul_op_common(const zdnn_ztensor *inputA,
 
   // Call zdnn_matmul_op on each tile.
   // Iterate over the tiles along the first dim of A.
+
+  // Commented out omp_set_nested because current build cannot guarantee that
+  // OpenMP is supported. Use "OMP_SET_NESTED=TRUE" at runtime instead.
+  // omp_set_nested(true);
+#pragma omp parallel for
```

Comment:
Have you tried to do `copyData, call zdnn op, copyData` in one single iteration? It would put data close to the current NNPA accelerator.

We have two kinds of parallelism: multiple CPUs and multiple NNPAs, and each one would require different number of threads. For example, we have more the number of CPUs than that of NNPAs. Does it make sense to define the number of NNPAs we'd like to use?

Reply:
This PR is just the initial enablement of omp in runtime library. We need to design a global control on how to use the thread resource.
One thing I tried is the different way of binding. According to document, we can use OMP_PLACES to describe the numa structure, and then use OMP_PROC_BIND to control the binding. For example, for parallel loop for CPU work, we may use `close` binding so that the threads can share cache. For zAIU threads we should use `spread` binding so that zAIU on different chip can be used. But my experiments did not go as I expected. Need further investigation.